examples/morphology/main.go

```diff
@@ -0,0 +1,35 @@
+// Port of http://members.shaw.ca/el.supremo/MagickWand/grayscale.htm to Go
+package main
+
+/*
+#cgo CFLAGS: -I/usr/local/include/ImageMagick-6
+#cgo LDFLAGS: -L/usr/sbin/lib
+*/
+
+import (
+	"fmt"
+    "gopkg.in/gographics/imagick.v2/imagick"
```

Comment:
Do you have a git hook you run for this? Actually just the exact command to run would be good - I though I had run it on all files, but apparently not.